apps/ledger-live-mobile/src/families/bitcoin/ConfirmationPostAlert.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Account } from "@ledgerhq/types-live";
+import React from "react";
+import { Trans } from "react-i18next";
+import Alert from "../../components/Alert";
+
+type Props = {
+  mainAccount: Account;
+};
+
+export default function ReceiveConfirmationAlertPost({ mainAccount }: Props) {
+  return (
+    <>
+      {mainAccount.currency.id === "dash" ? (
+        <Alert type="warning">
+          <Trans i18nKey="transfer.receive.receiveConfirmation.dashStakingWarning" />
+        </Alert>
+      ) : null}
```

Comment:
I'm noticing we don't display the taproot warning on mobile :thinking: I think we could add it with this PR, as it makes no sense that it's different from LLD.

Reply:
Yeah will do it